plugins/build-your-own.rst

```diff
@@ -4,7 +4,7 @@ Building Custom Plugins
 When writing your own Plugin, you need to be aware that the Plugin Client is async first.
 This means that every plugin must be written with Promises. More about this later.
 
-Each plugin must implement the ``Http\Client\Plugin\Plugin`` interface.
+Each plugin must implement the ``Http\Client\Common\Plugin`` interface.
```

Comment:
And what would be the format? Since from the documentation point of view we haven't added anything, but changed. Would it be "Since client-common 1.1 Plugins are part of that package" or "Prior to client-common 1.1 it was a separate package"?